### ISSUE TITLE
Mode problems

### ISSUE THREAD

**Issue by ue71603** [SIRI-CEN/SIRI#90]
Mode seems to be very unorganised within Siri:
We have:
* Mode based on AllModesEnumeration in siri_model/siri-reference.xsd which is clearly falsly defined
* ContinuosModesEnumeration in siri_model/siri_reference.xsd
* VehicleModesEnumeration in siri_model/siri_reference.xsd
*  Mode based on AffectedModeGroup which references VehicleMode, PtNetworkModeChoiceGoup and  AccessMode basedn on ifopt:AccessModesEnumeration 
* VehicleMode (siri_modes.xsd) is based on VehicleModesOfTransportEnumeration and contains a lot "blablablaService"
* In siri_modes we have also ModeChoiceGroup and PtModeChoiceGroup
* AccessModesEnumeration  is then again different fromn ContinuosModesEnumeration in ifopt/ifopt_modes.xsd

This needs to be simplified, it needs to be aligned with NeTEx. the blablaService needs to be dropped.

**Comment by Aurige** [comment]
Yes I agree... this is mainly due to the fact that SIRI started before NeTEx, tried to integrate values from TPEG and DATEX to ease compatibility, and the latest revision was finalised before NeTEx Part 5.
Also the way to clean it has to be discussed with the group since some decisions may raise backward compatibility issues.

**Comment by ue71603** [comment]
That's why it si an issue and not a pull request ;-). Do you bring it up at the next meeting? Or @haeckerbaer

**Comment by Aurige** [comment]
it can be mentioned but I'm afraid that we will need to wait for next revision

**Comment by TuThoThai** [comment]
Udpate as of 24 September 2025:
- Agreed to clarify Mode, Mode of operation, Transport Mode, Submode, etc.
- Change needs to reflect what has been changed in both Transmodel v6.2 and NeTEx v2.0
- The change will be done for v3.0
